tools/harness-thci/OpenThread.py

```diff
@@ -55,6 +55,8 @@ class OpenThread(IThci):
     isPowerDown = False              # indicate if Thread device experiences a power down event
     isWhiteListEnabled = False       # indicate if Thread device enables white list filter
     isBlackListEnabled = False       # indicate if Thread device enables black list filter
+    whiteList = set()                # cache whitelist devices when white list filter is enabled
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)
>Use one leading underscore only for non-public methods and instance variables.

Just suggestion: whitelist and blacklist are not phrase.
`whiteList` -> `_whitelist`
`blackList` -> `_blacklist`